crcsim/analysis.py

```diff
@@ -970,11 +983,11 @@ def compute_pop_rates(self, status_arrays: list):
         """
         # Sum all of the person status arrays to get an array of counts of the number of
         # people in each status for each year.
-        statuses: np.ndarray = sum(status_arrays)
+        status_array: np.ndarray = sum(status_arrays)
 
         # Convert to DataFrame so we can index by column name
         statuses = pd.DataFrame(
-            statuses,
+            status_array,
```

Comment:
This change is not relevant to the PR. It resolves a `mypy` error where `mypy` thought `statuses` was an ndarray instead of a DataFrame. I have no idea why this emerged now and not before - it may be worth digging into, but I went with a quick fix for now.

Reply:
I'm also a little surprised this wasn't flagged earlier, but I agree that renaming one of them is an appropriate fix.